tools/workbook-generator/data/.gitignore

```
@@ -0,0 +1 @@
+*/**journal
```

```suggestion
*/**journal

```
new line at eof to remove `-`